Automattic/Gravatar-SDK-iOS
# Localize more strings

Closes #

### Description

I went through the codebase looking for static strings that weren't localized yet.  Most of them I localized.  In `QuickEditor.swift` I just left a `TODO` item.

### Testing Steps

1. Open the Demo Project
2. Edit both the `UIKit` and `SwiftUI` schemes
3. For each scheme, go to `Run` --> `Options`, and enable `Show non-localized strings`
![Screenshot 2024-09-11 at 3 30 40 PM](https://github.com/user-attachments/assets/8745f13a-0e4c-4ee7-9bed-c1fb4ce61d5e)

With this enabled, all **_non-localized_** strings will be shown in ALL CAPITAL LETTERS.

- Smoke test `AvatarPickerView`
   - Attempt to upload photo
   - OBSERVE: the menu options "Choose a Photo" or "Take a Photo" appear as expected (not CAPITALIZED)
- Smoke test `Profile View`
   - OBSERVE: the `Palette` menu options' text appears as expected (not CAPITALIZED)
- Smoke test 'Profile editor with oauth`
   - OBSERVE: The `Done` button in the navigation bar appears as expected (not CAPITALIZED) 
![image](https://github.com/user-attachments/assets/6df7b0f0-b013-4014-9604-d1f95e044ade)
